docs/application/native/guides/internationalization/i18n.md

```diff
@@ -872,6 +876,63 @@ using a Number format for the PluralFormat locale.
 To add custom plural rules to the PluralFormat, create a Plural Rules object and add it in PluralFormat constructor.
 If you also pass a locale object to constructor, this locale will affect formatted number in the returned message texts.
 
+<a name="date_interval"></a>
+## Usages of date intervals
+
+The DateInterval API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) applications) allows you to [create date intervals](#date_interval_examples).
+
+DateInterval module represents time period between pair of `i18n_udate` values. This type contains
+number of milliseconds from 1 January 1970 00:00 UTC to desired date.
+
+### Format date intervals
+
+The DateIntervalFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) applications) allows you to [format created date intervals](#date_interval_examples).
+
+Purpose of this module is to enable formatting date intervals in a language-independent manner.Date Interval Format module formats a Date Interval into text as compactly as possible.
+For example, the date interval format from "Jan 11, 2008" to "Jan 18,. 2008" is
+"Jan 11-18, 2008" for English.
+
+There is no structural information in date time patterns. For any punctuations and
+string literals inside a date time pattern, we do not know whether it is just a separator,
+or a prefix, or a suffix. Without such information, so, it is difficult to generate
+a sub-pattern (or super-pattern) by algorithm. So, formatting a Date Interval is pattern-driven.
+
+A skeleton
```

Comment:
Suggestion:
Add a ":" after skeleton.
A skeleton:

Question:
Is it the skeleton of date time pattern?